src/client/models/bounces/Bounce.ts

```diff
@@ -1,12 +1,11 @@
 export interface Bounce {
-    RecordType: string;
     ID: number;
     Type: string;
     TypeCode: number;
     Name: string;
     Tag?: string;
     MessageID: string;
-    ServerID: string;
+    ServerID: number;
```

Comment:
Changed to match other references to ServerID.

[Example payload in documentation](https://postmarkapp.com/developer/webhooks/spam-complaint-webhook#spam-complaint-webhook-data) shows that this is a number rather than a string
```
{
  "RecordType": "Bounce",
  "MessageStream": "outbound",
  "ID": 4323372036854775807,
  "Type": "HardBounce",
  "TypeCode": 1,
  "Name": "Hard bounce",
  "Tag": "Test",
  "MessageID": "883953f4-6105-42a2-a16a-77a8eac79483",
  "Metadata" : {
    "a_key" : "a_value",
    "b_key": "b_value"
   },
  "ServerID": 23,
  "Description": "The server was unable to deliver your message (ex: unknown user, mailbox not found).",
  "Details": "Test bounce details",
  "Email": "john@example.com",
  "From": "sender@example.com",
  "BouncedAt": "2019-11-05T16:33:54.9070259Z",
  "DumpAvailable": true,
  "Inactive": true,
  "CanActivate": true,
  "Subject": "Test subject",
  "Content": "<Full dump of bounce>"
}
```

Reply:
hi @matthew-rollcredits 

thank you for this PR, I have one note here, removing RecordType from Bounce, Click, Open events is not correct. If you execute requests which use these interfaces, you will see that bounce, opens and clicks have record type set, through API too not just webhooks.

We will see to update developer docs to reflect that